mytimer/functions.py

```diff
@@ -313,6 +313,14 @@ def countdown_timer(
         time.sleep(max(0, 1 - (end - start)))
 
 
+def show_faces_list():
+    """Print all faces samples."""
+    for i, face in FACES_MAP.items():
+        print('Face %d' %(i))
+        tprint('\t\t\t\t  12 : 34 : 54 ', font=face)
+        print('='*130)
```

Comment:
You need to define and use string templates in `params.py`. Just like `show_programs_list`.
Not sure about that.
I suggest:
```python
        tprint('12 : 34 : 54', font=face)
        print('=' * 20)
```
what do you think @sepandhaghighi ?

Reply:
My suggestion :

```python
tprint('12 : 34 : 56', font=face)
print('=' * 80)
 ```
 
 1. Remove that `\t\t\t\t`
 2. Move `'12 : 34 : 56'` to `params.py` as `FACES_LIST_EXAMPLE_MESSAGE = "12 : 34 : 56"`